serde-generate/runtime/csharp/Serde.Tests/Serde.Tests.csproj

```diff
@@ -1,21 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2</TargetFramework>
-
-    <IsPackable>false</IsPackable>
-
+    <PropertyGroup>
+        <TargetFramework>netstandard1.6</TargetFramework>
```

Comment:
One of my earlier commits used this, but the test runner didn't support loading tests from netstandard DLLs. Can you verify that the tests are actually running?

Reply:
Good question.

Sadly, it turns out that on my laptop, nothing is run by `dotnet test` (disregarding this change). Testing in progress for CircleCI...

It looks like I should also add a Rust test to run the (non generated) C# unit tests.

Reply:
@mattico Oh no, this is bad: CI didn't catch the two errors I introduced even if I revert my first commit

Reply:
I can also confirm that C# tests aren't running on `master`. It appears the issue began on a5891983. It might be worthwhile to make the tests check that the test runner outputs `running <X> tests` where X > 0.

Reply:
https://github.com/novifinancial/serde-reflection/pull/77